Libs/MRML/Core/vtkMRMLPlotNode.h

```diff
@@ -0,0 +1,249 @@
+/*==============================================================================
+
+  Copyright (c) Kapteyn Astronomical Institute
+  University of Groningen, Groningen, Netherlands. All Rights Reserved.
+
+  See COPYRIGHT.txt
+  or http://www.slicer.org/copyright/copyright.txt for details.
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+  This file was originally developed by Davide Punzo, Kapteyn Astronomical Institute,
+  and was supported through the European Research Consil grant nr. 291531.
+
+==============================================================================*/
+
+#ifndef __vtkMRMLPlotNode_h
+#define __vtkMRMLPlotNode_h
+
+#include <string>
+#include <vector>
+
+// MRML Includes
+#include "vtkMRMLNode.h"
+
+class vtkMRMLStorageNode;
+class vtkMRMLTableNode;
+
+// VTK Includes
+class vtkColor4ub;
+class vtkPlot;
+class vtkTable;
+
+/// \brief MRML node to represent a Plot object
```

Comment:
It makes sense. Also the name will fit because the "Layout" properties are part of the View in a certain way. I like it and in this way we can remove one MRML class. I'll try and see if everything will work. Maybe for tomorrow afternoon I'll have the new version.